sbt-converter/src/main/scala/org/scalablytyped/converter/plugin/RemoteCache.scala

```diff
@@ -14,4 +17,19 @@ object RemoteCache {
     * The "directory" above will contain two folders after sync: `runs` and `org.scalablytyped`
     */
   case class Rsync(push: String, pull: URI) extends RemoteCache
+
+  case class S3(
+      push: S3.PushConfiguration,
+      pull: URI,
```

Comment:
can you infer the `pull` URI based on the S3 bucket?

Reply:
Only if we'd force to set the region (which by default in the client is optional as it can be defined in the systemwide aws config). And even then some more logic would be necessary e.g. for custom endpoints which might make it fragile. I'd rather let the user define it explicitly